pkgs/tools/text/rpl/default.nix

```diff
@@ -1,38 +1,48 @@
-{ lib, fetchFromGitHub, python3Packages }:
-
+{ lib, fetchPypi, fetchFromGitHub, python3Packages }:
+
+let
+  chainstream = python3Packages.buildPythonPackage rec {
+    pname = "chainstream";
+    version = "1.0.1";
+    pyproject = true;
+    nativeBuildInputs = [ python3Packages.setuptools ];
+    src = fetchPypi {
+      inherit pname version;
+      hash = "sha256-302P1BixEmkODm+qTLZwaWLktrlf9cEziQ/TIVfI07c=";
+    };
+  };
+in
 python3Packages.buildPythonApplication rec {
   pname = "rpl";
-  version = "1.10";
-
-  # Tests not included in pip package.
-  doCheck = false;
+  version = "1.15.5";
 
+  pyproject = true;
 
-  src = fetchFromGitHub {
-    owner = "rrthomas";
-    repo = "rpl";
-    rev = "4467bd46a7a798f738247a7f090c1505176bd597";
-    sha256 = "0yf3pc3fws4nnh4nd8d3jpglmsyi69d17qqgpcnkpqca5l4cd25w";
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-rhPS+hwbjqq3X/V1bL6pzGg2tVxBkeMyUhaCvmneG4M=";
   };
 
-  patches = [
-    ./remove-argparse-manpage.diff # quickfix for ImportError: No module named build_manpages.build_manpages
+  nativeBuildInputs = [
+    python3Packages.setuptools
+  ];
+
+  nativeCheckInputs = [
+    python3Packages.pytest-datafiles
+    python3Packages.pytest
   ];
 
   propagatedBuildInputs = [
-    #python3Packages.argparse-manpage # TODO
+    chainstream
+    python3Packages.argparse-manpage
     python3Packages.chardet
+    python3Packages.regex
   ];
 
-  installPhase = ''
-    mkdir -p $out/bin
-    mv rpl $out/bin
-  '';
-
   meta = with lib; {
     description = "Replace strings in files";
-    homepage    = "https://github.com/rrthomas/rpl";
-    license     = licenses.gpl2;
+    homepage = "https://github.com/rrthomas/rpl";
+    license = licenses.gpl2;
     maintainers = with maintainers; [ teto ];
```

Comment:
do you want to set yourself as maintainer ? I dont use it anymore

Reply:
Yes, sounds good to me :+1: